ext/handlers_http.h

```diff
@@ -180,6 +180,7 @@ static inline void ddtrace_inject_distributed_headers_config(zend_array *array,
 
                 zend_string *full_tracestate = ddtrace_format_tracestate(tracestate, origin, sampling_priority, propagated_tags, tracestate_unknown_dd_keys);
                 if (full_tracestate) {
+                    // Replace dd=.... with dd=p:<zero padded span_id>;.....
```

Comment:
Here I am avoiding updating  `ddtrace_format_tracestate`. We only NEED to add `p:........` in the tracestate when generating distributed tracing headers (ex: we do not need this field in span_links).